### PR TITLE
RDM-7120: Regression: Unable to create Draft SDO in DEMO

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-7120

### Change description ###

Change select element binding to [(ngModel)] and use [ngValue] instead of formControl and "value" attributes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts [hunk 1]
```diff
@@ -7,18 +7,10 @@ import { AbstractFieldWriteComponent } from '../base-field/abstract-field-write.
...
-    if (this.hasSanitisedDynamicListData()) {
-      this.fixedListControl.setValue(this.caseField.value);
+    let isNull = this.caseField.value === undefined || this.caseField.value === '';
+    if (isNull) {
+      this.caseField.value = null;
     }
   }
-
-  private hasSanitisedDynamicListData() {
-    return this.fixedListControl.value && this.fixedListControl.value.value;
```
Comment (mario-paniccia) [inline src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts]: this line seems redundant to me so it should be ok to remove. But just for piece of mind can we verify dynamic lists are still working?

File: src/shared/components/palette/fixed-list/write-fixed-list-field.html [hunk 2]
```diff
@@ -1,16 +1,14 @@
-<div class="form-group" [ngClass]="{'form-group-error': !fixedListControl.valid && fixedListControl.touched}">
+<div class="form-group" [ngClass]="{'form-group-error': false}">
```
Comment (mario-paniccia) [inline src/shared/components/palette/fixed-list/write-fixed-list-field.html]: do we need `[ngClass]="{'form-group-error': false}"` at all now?

Comment (kiran-yenigala-hmcts) [reply src/shared/components/palette/fixed-list/write-fixed-list-field.html]: Removed

File: src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts]: ok.. Will ask Ashley to test dynamic lists as well to see this change

File: src/shared/components/palette/fixed-list/write-fixed-list-field.html [hunk 3]
```diff
@@ -1,16 +1,14 @@
-<div class="form-group" [ngClass]="{'form-group-error': !fixedListControl.valid && fixedListControl.touched}">
+<div class="form-group">
 
   <label [for]="id()">
     <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
     <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message" *ngIf="fixedListControl.errors && fixedListControl.touched">{{fixedListControl.errors | ccdFirstError}}</span>
```
Comment (mario-paniccia) [inline src/shared/components/palette/fixed-list/write-fixed-list-field.html]: we have removed `ccdFirstError` pipe. If you look at FirstErrorPipe, this pipe is used to provide common standard error messages. For example if the field is required, the common error message should be:
`'This field is required'`
What error message are we getting now?

Comment (mario-paniccia) [reply src/shared/components/palette/fixed-list/write-fixed-list-field.html]: we need to ask QAing around error messages

**Pull request merged**